cognite/client/data_classes/workflows.py

```diff
@@ -708,9 +719,9 @@ class WorkflowVersionList(CogniteResourceList[WorkflowVersion]):
 
     _RESOURCE = WorkflowVersion
 
-    def as_ids(self) -> WorkflowIds:
-        """Returns a WorkflowIdList of workflow version ids for the workflow versions in the list."""
-        return WorkflowIds([workflow_version.as_id() for workflow_version in self.data])
+    def as_ids(self) -> _WorkflowIds:
+        """Returns a WorkflowIds object with the workflow version ids."""
+        return _WorkflowIds([workflow_version.as_id() for workflow_version in self.data])
```

Comment:
I guess this has been a discussion that I've missed - but - why are we returning an instance of a private class `(_WorkflowIds`) from a public method?

Reply:
Good point, made it public again. And added some validation to it.